src/nxdoc/nuxeo-server/administration/database-configuration/documentdb.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Amazon DocumentDB
+labels:
+    - database
+    - documentdb
+tree_item_index: 700
+
+---
+The Nuxeo Platform supports Amazon DocumentDB
+
+## Configuration
+
+Since Nuxeo 2023.15, using the `mongodb` template, you can use `DocumentDB` as a `DBS` solution with some limitations:
+
+- fulltext is not supported as `DocumentDB` has various limitations in that aspect (english only, no term exclusion, semantics differences...) see [Differences With MongoDB](https://docs.aws.amazon.com/documentdb/latest/developerguide/text-search.html#text-index-mongo-diff) for more details.
```

Comment:
fulltext search is indeed supported by Nuxeo since fulltext is extracted and searchable using elastic,
Only the fulltext search is not supported at repository level because of DocumentDB limitation